panther_analysis_tool/main.py

```diff
@@ -383,24 +385,11 @@ def upload_zip(
 
                 else:
                     logging.warning("Exhausted retries attempting to perform bulk upload.")
-                    logging.error("Failed to upload to backend: %s", convert_unicode(be_err))
-                    return_code = 1
-                    return_archive_fname = ""
-                    break
+                    return 1, f"Failed to upload to Panther: {err}"
 
             # PEP8 guide states it is OK to catch BaseException if you log it.
             except BaseException as err:  # pylint: disable=broad-except
-                logging.error("failed to upload to backend: %s", err)
-                return_code = 1
-                return_archive_fname = ""
-                break
-
-    if return_code != 0:
-        return return_code, return_archive_fname
-
-    return_code, _ = panthersdk_upload.run(backend=backend, args=args, indirect_invocation=True)
-
-    return return_code, return_archive_fname
+                return 1, f"Failed to upload to Panther: {err}"
```

Comment:
do we not want to log the error here still?

Reply:
The returned string is logged as an error. So we can return an error instead of logging it